[2023-06-23 20:57:17,976] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2023-06-23T20:56:55.546198+00:00 [None]>
[2023-06-23 20:57:18,029] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2023-06-23T20:56:55.546198+00:00 [None]>
[2023-06-23 20:57:18,031] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-06-23 20:57:18,032] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2023-06-23 20:57:18,033] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-06-23 20:57:18,086] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_zero> on 2023-06-23T20:56:55.546198+00:00
[2023-06-23 20:57:18,095] {standard_task_runner.py:52} INFO - Started process 1584 to run task
[2023-06-23 20:57:18,112] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_zero', '2023-06-23T20:56:55.546198+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmp6bv5pg3k', '--error-file', '/tmp/tmpo02fka8h']
[2023-06-23 20:57:18,115] {standard_task_runner.py:77} INFO - Job 7: Subtask task_zero
[2023-06-23 20:57:18,250] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_zero 2023-06-23T20:56:55.546198+00:00 [running]> on host 7f7a4865b07a
[2023-06-23 20:57:18,417] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_zero
AIRFLOW_CTX_EXECUTION_DATE=2023-06-23T20:56:55.546198+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-23T20:56:55.546198+00:00
[2023-06-23 20:57:18,422] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2023-06-23 20:57:18,425] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'pip install beautifulsoup4']
[2023-06-23 20:57:18,447] {subprocess.py:75} INFO - Output:
[2023-06-23 20:57:26,828] {subprocess.py:79} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-06-23 20:57:27,900] {subprocess.py:79} INFO - Requirement already satisfied: beautifulsoup4 in /home/***/.local/lib/python3.6/site-packages (4.12.2)
[2023-06-23 20:57:27,943] {subprocess.py:79} INFO - Requirement already satisfied: soupsieve>1.2 in /home/***/.local/lib/python3.6/site-packages (from beautifulsoup4) (2.3.2.post1)
[2023-06-23 20:58:16,179] {subprocess.py:83} INFO - Command exited with return code 0
[2023-06-23 20:58:16,325] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_zero, execution_date=20230623T205655, start_date=20230623T205717, end_date=20230623T205816
[2023-06-23 20:58:16,454] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-23 20:58:16,496] {local_task_job.py:151} INFO - Task exited with return code 0
